README.rst

```diff
@@ -375,6 +375,55 @@ Usage exmaple:
     >>> user.organizations_dict.keys()
     ... dict_keys(['20135c30-d486-4d68-993f-322b8acb51c4'])
 
+Organization membership permission_classes
+------------------------------------------
```

Comment:
I would change this title to something like `Django REST Framework Permission Classes`